AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/Dependency/Pinpoint/PinpointContext.swift

```diff
@@ -5,12 +5,46 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+import Amplify
+import AWSClientRuntime
 import AWSPinpoint
+import AWSPluginsCore
 import Foundation
 
+protocol UserDefaultsBehaviour {
```

Comment:
Do you know what the protocol naming convention is for this project? is it `UserDefaultsProtocol`, `UserDefaultsBehavior` or `UserDefaultsBehaviour`? I've seen a mix so far, we should pick one and use it consistently.